src/domain/collection/curatedEventList/utils.ts

```diff
@@ -0,0 +1,104 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { toast } from 'react-toastify';
+
+import {
+  EventFieldsFragment,
+  LinkedEventsSource,
+  useEventsByIdsQuery,
+} from '../../../generated/graphql';
+import { isEventClosed } from '../../event/EventUtils';
+import { EventType } from '../../event/types';
+
+export const PAGE_SIZE = 10;
```

Comment:
exported but only used locally

Reply:
Seems to be duplicated, defined in two places